server/test/tp/files/hg38/TermdbTest/mutationpersample/table

```diff
@@ -0,0 +1,12 @@
+91	sdhanda/mb_portal/BT_database/TermdbTest/mutationpersample/91
```

Comment:
i don't think this `table` file is needed

Reply:
ok thanks removed.